ee-feature-pack/galleon-common/src/main/resources/feature_groups/embedded-activemq.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<feature-group-spec name="embedded-activemq" xmlns="urn:jboss:galleon:feature-group:1.0">
```

Comment:
For ee-9 module, we are also generating a configuration that includes the embedded messaging broker. Together with this PR, you should also adjust the ee-9 module to reuse this feature group spec and avoid re-defining it there.

See: https://github.com/wildfly/wildfly/blob/8958a48418159da5b0e3fce03aeb9f211342f2b1/ee-9/galleon-content/src/main/resources/packages/docs.examples.configs/pm/wildfly/tasks.xml#L23-L26